web/src/features/exchanges/ExchangeArrow.tsx

```diff
@@ -64,6 +66,11 @@ function ExchangeArrow({
     r: rotation + (netFlow > 0 ? 180 : 0),
   };
 
+  let tooltipClassNameTest = 'max-h-[256px] max-w-[512px] top-[-76px] hidden md:flex';
```

Comment:
Why is this named `tooltipClassNameTest`?

I don't think it should have test in the name?